.eslintrc.js

```diff
@@ -0,0 +1,250 @@
+module.exports = {
+  'plugins': [
```

Comment:
If you remove this plugin, you can also remove the `eslint-plugin-html` dependency.

Reply:
NIce